src/backend/src/prisma/migrations/20230214001747_add_wp_stage_and_tasks/migration.sql

```diff
@@ -5,7 +5,7 @@ CREATE TYPE "Task_Priority" AS ENUM ('LOW', 'MEDIUM', 'HIGH');
 CREATE TYPE "Task_Status" AS ENUM ('IN_BACKLOG', 'IN_PROGRESS', 'DONE');
 
 -- CreateEnum
-CREATE TYPE "Work_Package_Stage" AS ENUM ('RESEARCH', 'DESIGN', 'MANUFACTURING', 'INTEGRATION');
+CREATE TYPE "Work_Package_Stage" AS ENUM ('RESEARCH', 'DESIGN', 'MANUFACTURING', 'INSTALL');
```

Comment:
you'll wanna make this a new migration that adds the Install Type, converts any 'INTEGRATION' instances to 'INSTALL', then removes the install type

Reply:
I would start this process by reverting the change in this file, saving, and then doing yarn prisma:migrate which should generate a new migration that adds the INSTALL value for the stage enum. You can then edit it to convert 'integration' instances to 'install' instances and then add a line to remove the 'integration type'.

This is gonna require some SQL research, and should be tested thoroughly locally